### PR TITLE
Add resident upload and new village form

### PR DESCRIPTION
**Pull request by daniellee0**

### CODE REVIEW

File: src/AddResident.js [hunk 1]
```diff
@@ -32,6 +37,15 @@ export default class AddResident extends Component {
                             <span aria-hidden="true">&times;</span>
                         </button>
                     </div> : <div/>}
+                {this.state.uploadedResidents === true ?
+                  <div className="alert alert-success alert-dismissible fade show" role="alert">
+                      Successfully uploaded file!
+                      <button type="button" className="close" aria-label="Close" onClick={() => {
+                          this.setState({uploadedResidents: false})
+                      }}>
+                          <span aria-hidden="true">&times;</span>
```
Comment (kushaljh) [inline src/AddResident.js:45]: Why do we need this hidden span?

And we have a similar one above

File: src/AddResident.js [hunk 2]
```diff
@@ -274,9 +289,53 @@ class AddResidentForm extends Component {
                            onChange={this.handleChange}/>
                 </div>
             </div>
-            <Button style={{margin: "15px"}} size="md" type="submit" className="btn btn-primary" value="Submit">Add
+            <Button style={{margin: "15px"}} size="md" type="submit" className="btn btn-primary" value="Submit">Submit
```
Comment (kushaljh) [inline src/AddResident.js]: Why submit?

Add is good imo

**Review by wilsmic2 — DISMISSED** [review]

File: src/IncidentReport.js [hunk 3]
```diff
@@ -171,10 +171,9 @@ class IncidentReport extends Component {
                 .then((text) => {
                     alert(text.error);
                 });
-            } else {
+            } else if (response.status === 201) {
               form.reset();
               update(true);
```
Comment (aashnasheth) [inline src/IncidentReport.js:176]: What does 201 signify? Also, is update(true) what you meant? Not this.updateIncidentStatus(true)?

File: src/App.js [hunk 4]
```diff
@@ -53,21 +55,27 @@ class HomePage extends Component {
...
+                            <span>Add an Incident</span>
                         </Button>
                     </Link>
-                    <Link to="/residentdirectory">
+                    <Link style={{margin: "30px"}} to="/residentdirectory">
                         <Button  size="lg">
                             <span>Resident Directory</span>
                         </Button>
                     </Link>
+                    <Link style={{margin: "30px"}} to="/addVillage">
```
Comment (wilsmic2) [inline src/App.js]: should Village be capitalized in the /addVillage?

File: src/AddResident.js (hunk 1 above)
Comment (daniellee0) [reply src/AddResident.js:45]: This span is necessary to display the "x" button that closes the alert.

File: src/AddVillage.js [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+                    </div> : <div/>}
+                <div style={{ //title div
+                    display: "flex",
+                    justifyContent: "center",
+                    alignItems: "center"
+                }}>
+                    <h1>Add Village</h1>
+                </div>
+                <label>List of Villages</label>
+                  <Select style={{marginBottom: "30px"}} value={this.state.village} id="getVillage" className="dropdown" onChange={(option) => {this.handleDropdownMulti(option, "village")}} name="village"
```
Comment (wilsmic2) [inline src/AddVillage.js]: Cannot find handleDropdownMulti

**Review by wilsmic2 — APPROVED** [review]

**Review by kushaljh — APPROVED** [review]

**Pull request merged**